.github/workflows/ci.yml

```
@@ -17,18 +18,24 @@ env:
 
 jobs:
   # 🔍 코드 품질 검사 (빠른 피드백)
+  quality-check:
```

actionlint (1.7.7)

"steps" section is missing in job "quality-check"

(syntax-check)